examples/service-catalog/service-catalog.yaml

```diff
@@ -0,0 +1,295 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: service-catalog
+objects:
+
```

Comment:
you mean the newlines?  yeah, i have it broken up for easier management/editing at the moment, but i can remove the newlines before the merge.